deploy/images/README.md

```diff
@@ -8,7 +8,7 @@ In the current folder, type `make` and the images are created in order.
 
 It takes some time.
 
-Later on, if the `crg/ega-*` do need to be recreated, one can type `make`.
```

Comment:
This commit contains other changes that does not match the commit message,
these should be split out to a separate commit to minimise confusion.